### ISSUE TITLE
tlg0057.tlg096 is currently part of tlg0057.tlg095

### ISSUE THREAD

**Issue by AlisonBabeu** [OpenGreekAndLatin/First1KGreek#837]
So as I've been working my way through the Galen repository I've been checking any works that it seems we don't have.

The TLG identifies a work called "De humero iis modis prolapso quos Hippocrates non vidit" with the canonical ID of tlg0057.tlg0096. We aren't missing this work it turns out because it has been included in the [tlg0057.tlg0095](https://github.com/OpenGreekAndLatin/First1KGreek/blob/master/data/tlg0057/tlg095/tlg0057.tlg095.1st1K-grc1.xml) file and is  separated out starting at line 706 with: 
`<div type="textpart" subtype="chapter" n="excursus">`

According to the TLG, work tlg0057.tlg0095 "In Hippocratis librum de articulis et Galeni in eum commentarii iv" is on Kuhn's volume 18.1,  pgs. 300‑345 and 423‑767.  Work tlg0057.tlg0096 is on pages 346-422. My main question is, should we separate this work out and add it into the Galen repository separately with a separate ID?

**Comment by Pria20** [comment]
Should we still take this text (096) out and add it as a new file into the Galen repository? We could.

**Comment by annettegessner** [comment]
Yes, I would suggest you create a new file (0096) and check, if it's better to take this part out of the other file (0095) (I would assume this the right approach) or to leave it in and just copy it! Thanks!

**Comment by sonofmun** [comment]
Yes, take the text from line 706 and make a new file, `tlg0057.tlg096.1st1K-grc1.xml` out of it. Make sure to copy the teiHeader (with the appropriate changes) and to create the work level `__cts__.xml` file for this work.

**Comment by LindaMcNulty** [comment]
I will work on this issue next.

**Comment by LindaMcNulty** [comment]
I have made all the necessary changes and solved this issue in my pull request #1547
